modelling/src/main/java/org/axonframework/modelling/command/AnnotationCommandTargetResolver.java

```diff
@@ -44,6 +49,20 @@
  */
 public class AnnotationCommandTargetResolver implements CommandTargetResolver {
 
+	private Class<? extends Annotation> identifierAnnotation = TargetAggregateIdentifier.class;
+	private Class<? extends Annotation> versionAnnotation = TargetAggregateVersion.class;
+
+	public static final Builder builder() {
+		return new Builder();
+	}
+
+	/**
+	 * Default settings. Use {@link #builder()} for custom setup.
+	 */
+	public AnnotationCommandTargetResolver() {
```

Comment:
As commented above, I’m not sure about that. I’ve seen this class been instanciated in many places in axon modules. I’d prefer smaller, non breaking changes over the most perfectly safe builder. I will analyze the places that would break and write again.

Reply:
I'll leave the constructor and annotate it as deprecated. I'll leave it up to you, to delete it, if you want.

Reply:
Deprecating the original is a good idea, great thinking :+1: